ReactViewControl/ReactViewRender.LoaderModule.cs

```diff
@@ -137,13 +132,13 @@ private class SyncFunction {
                 ViewRender.WebView.ExecuteScript($"import('{loaderUrl}').then(m => m.default.{LoaderModuleName}).then({LoaderModuleName} => {LoaderModuleName}.{functionName}({string.Join(",", args)}))");
             }
 
-            private static string SerializeComponent(IViewModule component, bool forceNativeSyncCalls) {
+            private static string SerializeComponent(IViewModule component) {
                 var nativeObjectMethodsMap = component.Events
-                    .Select(g => new KeyValuePair<string, object>(g, forceNativeSyncCalls ? SyncFunction.Instance : JavascriptSerializer.Undefined))
+                    .Select(g => new KeyValuePair<string, object>(g, JavascriptSerializer.Undefined))
```

Comment:
Why don't we need this "forceNativeSyncCalls" anymore?

Reply:
that mechanism had problems and I removed